src/main/java/address/browser/AddressBookPagePool.java

```diff
@@ -0,0 +1,119 @@
+package address.browser;
+
+import address.browser.page.GithubProfilePage;
+import address.model.datatypes.Person;
+import address.util.FxViewUtil;
+import com.teamdev.jxbrowser.chromium.Browser;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Manages a pool of activePages, where each has a non-identical content.
+ *
+ * To ensure correct functionality and consistency of the pool of pages:
+ * 1) To use loadPersonPage(), calling clearPagesNotRequired() or clearPageNotRequired() first is required.
+ *    - Call clearPageNotRequired(parameter) when only a single person page (non-identical with the pool of pages)
+ *      is to be loaded using the loadPersonPage() method.
+ *    - Call clearPagesNotRequired(parameter) when multiple person page (non-identical with the pool of pages)
+ *      are to be loaded using the loadPersonPage() method.
+ */
+public class AddressBookPagePool {
+
+    private ArrayList<GithubProfilePage> activePages;
+
+    /**
+     * For recycling of browser instance.
+     */
+    private Stack<EmbeddedBrowser> inActiveBrowserStack;
```

Comment:
Does this mean the stack is active? `inActiveBrowserStack` -> `in active browser stack` by camel casing. If meant to be inactive stack, `inactiveBrowserStack`.